Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoViewController.swift

```diff
@@ -125,7 +125,21 @@ final class CallInfoViewController: UIViewController, CallActionsViewDelegate, C
         [statusViewController.view, accessoryViewController.view, actionsView].forEach(stackView.addArrangedSubview)
         statusViewController.didMove(toParent: self)
     }
-
+    
+    private func showMutedToastMessageIfNeeded() {
+        if case .established(let duration) = configuration.state,
+            duration <= 5.0,
+            configuration.isMuted,
+            !hasMutedToastBeenShown {
+            let toastConfig = ToastConfiguration(message: L10n.Localizable.Call.Toast.MutedOnJoin.message,
+                                                 colorScheme: ColorSchemeColor.utilityNeutral,
+                                                 variant: ColorSchemeVariant.light,
+                                                 dismissable: true, moreInfoAction: nil)
```

Comment:
You may be able to make this a little more readable by using type inference:

```suggestion
            let toastConfig = ToastConfiguration(message: L10n.Localizable.Call.Toast.MutedOnJoin.message,
                                                 colorScheme: .utilityNeutral,
                                                 variant: .light,
                                                 dismissable: true, 
                                                 moreInfoAction: nil)
```